src/js/api/brand.js

```diff
@@ -11,6 +11,45 @@ if (
     var ripe = base.ripe;
 }
 
+/**
+ * Returns the logo of a brand.
+ *
+ * @param {Object} options A map with options, such as:
+ *  - 'brand' - The brand of the model
+ *  - 'version' - The version of the build, defaults to latest
+ *  - 'variant' - The variant of the logo, that controls semantics of the logo
+ *  - 'format' - The format of the logo image to be retrieved defaults to png
+ *  - 'size' - The size of the logo image
+ * @param {Function} callback Function with the result of the request.
+ * @returns {XMLHttpRequest} The brand's logo.
+ */
+ripe.Ripe.prototype.getLogo = function(options, callback) {

```

Comment:
Add an equivalent to `getMeshUrl` as well, as it allows more flexible use-cases.

Reply:
Nevermind, I just noticed it already exists.